localization/react-intl/src/app/components/feed/FeedBlankState.json

```diff
@@ -0,0 +1,27 @@
+[
+  {
+    "id": "feedBlankState.title",
+    "description": "Title displayed on feed page when there are no fact-checks to be listed.",
+    "defaultMessage": "No fact-checks"
+  },
+  {
+    "id": "feedBlankState.description",
+    "description": "Message displayed on feed page when there are no fact-checks to be listed.",
+    "defaultMessage": "Try {resetLink} to find fact-checks or verify that the {listLink} has items"
```

Comment:
I'm not sure if this works well for translation, but it's helpful to provide more details - such as a fully readable sentence, or possible variable values - in the description when interpolating not so trivial pieces of content like this.

Reply:
Makes sense! Fixed here: https://github.com/meedan/check-web/commit/a38c8ccb35f6e200d54dd90951b92f33b363d123